sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/DeclarativeConfigContext.java

```diff
@@ -39,6 +42,15 @@ List<Closeable> getCloseables() {
     return Collections.unmodifiableList(closeables);
   }
 
+  @Nullable
+  public MeterProvider getMeterProvider() {
+    return meterProvider;
+  }
+
+  public void setMeterProvider(MeterProvider meterProvider) {
+    this.meterProvider = meterProvider;
```

Comment:
any chance this needs to be cross-thread safe? In which case, putting a memory barrier around the assignment would be necessary.

Reply:
no, this doesn't need to be thread safe - it belongs to the  set up logic that is done in a single thread at the start